src/usecases/refreshToken.js

```diff
@@ -0,0 +1,24 @@
+import moment from "moment";
+
+export default ({ getTokenProvider }) => (token) => {
+  const expiresAt = new Date(token.exp * 1000);
+  if (
+    moment(moment()).isBetween(
```

Comment:
Do we need to wrap `moment` in `moment`? 🤔

Reply:
Nope 👍